tests/manage/mcg/test_bucket_policy.py

```diff
@@ -985,62 +986,3 @@ def test_multipart_with_policy(self, mcg_obj, bucket_factory):
         # Completing the Multipart Upload
         logger.info(f"Completing the MP Upload with on bucket: {bucket}")
         complete_multipart_upload(obc_obj, bucket, object_key, upload_id, uploaded_part)
-
```

Comment:
@sagihirshfeld I think you mistakenly removed this test definition. Could you please add it back